site/docs/components/switch/accessibility.mdx

```diff
@@ -0,0 +1,35 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+- On focus, the screen reader should:
+  - Identify the switch
```

Comment:
```suggestion
  - Identify the Switch
```

Reply:
keeping it lowercase for now as per content team's guidance